src/Core/Security/ProviderConfiguration/Application/OpenId/UseCase/FindOpenIdConfiguration/FindOpenIdConfigurationResponse.php

```diff
@@ -184,4 +180,20 @@ public static function authorizationRulesToArray(array $authorizationRules): arr
             ];
         }, $authorizationRules);
     }
+
+    /**
+     * @param AuthenticationConditions $authenticationConditions
+     * @return array<string,bool|string|string[]>
+     */
+    public static function authenticationConditionsToArray(AuthenticationConditions $authenticationConditions): array
```

Comment:
You have the same function into "DbWriteOpenIdConfigurationRepository.php" file.
Maybe you could put this method iside "AuthenticationConditions" class ?

Reply:
agreed but not in the Model as it is not related to business logic but presentation, we'll discuss about it !